js/index.js

```diff
@@ -40,3 +40,75 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let imgCta = document.getElementById("cta-img")
+imgCta.setAttribute('src', siteContent["cta"]["img-src"])
+
+let imgMiddle = document.getElementById("middle-img")
+imgMiddle.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+// //update content
```

Comment:
I love how you have your code organized here. Very clean and easy to read/understand. Great job